### DISCUSSION TITLE
Liquid pressure problem

### DISCUSSION THREAD

**Discussion by hekungeo** [Anura3D/Anura3D_OpenSource#16] [Q&A] [answered]
Hi, Dear all,

I am processing the seepage analysis using Anura 2022. A question about liquid pressure confuses me. In a slope model, the liquid pressure refers to pore water pressure or not? Generally, the pore water pressure should be negative above the groundwater level as will increase with the rainfall infiltration, and then become positive, further inducing the slope failure. 

However, in my case and other literature, the liquid pressure seems positive above and negative below the groundwater level. as following:
![image](https://user-images.githubusercontent.com/105496180/175255776-6bfec0a1-ae49-435e-8025-909cf5b68e78.png)

and here's a figure in the paper: Cuomo, S., Di Perna, A., & Martinelli, M. (2021). Modelling the spatio-temporal evolution of a rainfall-induced retrogressive landslide in an unsaturated slope. Engineering Geology, 294, 106371.
![image](https://user-images.githubusercontent.com/105496180/175256058-cae43696-077c-4031-8ca7-a756b4700504.png)

Could you please help me with this?

Thx in advance.

Cheers,
Kun

**Comment by ayerrocolom** [comment, marked as answer]
Hello,
Liquid pressure is the pressure of the water. The outputs in Anura3D follow a sign conversion that is positive for traction and negative for compression; so suction is positive.
Best,
Alba

**Reply by hekungeo** [reply]
Hi Alba,

Thank you for your explanation. So, it means the suction. S=Ugas-Uwater. While Uwater is negative, Ugas=0 for two-phase simulation, the suction would be positive. 

Thanks a lot.

Best,
Kun

**Reply by ayerrocolom** [reply]
Yes, that's correct.

**Reply by CPLA81192** [reply]
I'm very sorry to bother you. When simulating rainfall infiltration, when the initial liquid pressure is high, such as 2000kPa, the soil is prone to expansion and loosening, and it is easy to disconnect from the bottom. May I ask what may be the reason for this situation
![1705366603569](https://github.com/Anura3D/Anura3D_OpenSource/assets/97551917/855f8934-6af1-46ed-b39e-14f8c16f2a77)
